_opensearch/rest-api/create-index.md

```diff
@@ -0,0 +1,155 @@
+master_timeout | Time | How long to wait for a connection to the master node. Default `30s`.
+timeout | Time | How long to wait for the request to return. Default `30s`.
+
+## Request body
+
+As part of your request, you can supply parameters in your request's body that specify index settings, mappings, and aliases for your newly created index. The following sections provide more information about index settings and mappings. An alias is a virtual index name that references one or more indices. Because changing an alias's reference index is an atomic operation, aliases offer the benefit of reindexing data without any downtime. See [Index Aliases](../../index-alias) to learn more.
+
+### Index settings
+
+Index settings are separated into two varieties: static index settings and dynamic index settings. Static index settings are settings you specify and index creation and cannot be changed once the index is created. Dynamic index settings can be specified or changed at any time, including index creation.
+
+#### Static index settings
+
+Setting | Description
+:--- | :---
+index.number_of_shards | The number of primary shards in the index. (Default: 1)
+index.number_of_routing_shards | The number of routing shards used to split an index.
+index.shard.check_on_startup | Whether the index's shards should be checked for corruption. Available options are `false` (do not check for corruption), `checksum` (check for physical corruption), and `true` (check for both physical and logical corruption). (Default: `false`)
+index.codec | The compression type to use to compress stored data. Available values are `best_compression` and `default`.
+index.routing_partition_size | The number of shards a custom routing value can go to. Routing helps an out-of-balance by relocating values to a subset of shards rather than just a single shard. To enable, set this value to greater than 1 but less than `index.number_of_shards`. (Default: 1)
+index.soft_deletes_enabled | Whether to retain a history of operations performed on the index.
+index.soft_deletes.retention_lease.period | The maximum amount of time to retain a shard's history of operations. (Default: `12h`)
+index.load_fixed_bitset_filters_eagerly | Indicates whether cached filters should be pre-loaded for nested queries. Available options are `true` and `false`. (Default: true)
+index.hidden | Whether the index should be hidden. Hidden indices are not returned as part of queries that have wildcards. Available options are `true` and `false`. (Default: `false`)
+
+#### Dynamic index Settings
+
+Setting | Description
+:--- | :---
+index.number_of_replicas | The number of replica shards each shard should have. (Default: 1)
+index.auto_expand_replicas | Automatically expands the number of replicas based on the amount of data nodes in the cluster. Specify a lower bound and upper limit (for example, 0-9), or `all` for the upper limit. (Default: disabled)
```

Comment:
Provide an example here. "For example, if your cluster has six nodes and you set X, Y happens."